extensions/src/main/java/org/pentaho/platform/plugin/action/ActionRunner.java

```diff
@@ -68,14 +70,17 @@ public ActionRunner( final IAction actionBean, final String actionUser, final Ma
   }
 
   public Boolean call() throws ActionInvocationException {
+    final String workItemUid = ActionUtil.extractUid( params );
     try {
-      return callImpl();
+      final boolean result = callImpl();
+      WorkItemLifecyclePublisher.publish( workItemUid, params, WorkItemLifecyclePhase.SUCCEEDED );
```

Comment:
@pedrofvteixeira I was not aware of this conversation. If this is a requirement, we can certainly do the work, but I agree with@pentaho-esager that this is out of the scope of this PR